pkg/kubectl/cmd/cmd.go

```diff
@@ -159,6 +159,16 @@ __kubectl_require_pod_and_container()
     return 0
 }
 
+# Print serviceaccounts across all namespaces in the format "<namespace>:<name>"
+__kubectl_get_serviceaccounts_all_namespaces()
+{
+    local template kubectl_out
+    template="{{ range .items }}{{ .metadata.namespace }}:{{ .metadata.name }} {{ end }}"
+    if kubectl_out=$(kubectl get -o template --template="${template}" serviceaccount --all-namespaces 2>/dev/null); then
```

Comment:
getting across all namespaces isn't something I'd expect to be part of an autocomplete... it's not consistently available and retrieves a very large amount of data on large clusters